databricks/koalas/metadata.py

```diff
@@ -30,69 +32,189 @@ class Metadata(object):
     """
     Manages column names and index information.
 
-    :ivar _column_fields: list of the Spark field names to be seen as columns in Koalas DataFrame.
-    :ivar _index_info: list of pair holding the Spark field names for indexes,
+    :ivar _data_columns: list of the Spark field names to be seen as columns in Koalas DataFrame.
```

Comment:
This is super useful doc, thank you for your time writing it. Can you add a comment to say this is an internal class and that it is not meant to be directly accessed by the users (only developers)?

Reply:
sure